erts/doc/src/erlang.xml

```diff
@@ -9053,6 +9053,18 @@ ok
               <seealso marker="doc/system_principles:versions">
               System principles</seealso> in System Documentation.</p>
           </item>
+          <tag><marker id="system_info_otp_minor_version"/>
+            <c>otp_minor_version</c></tag>
+          <item>
+            <marker id="system_info_otp_minor_version"></marker>
```

Comment:
Unnecessary duplicate id (declared with the same value as prior <marker />

Reply:
Will fix up

Reply:
@paulo-ferraz-oliveira Should I go ahead an fix up the entry for otp_release? It had a duplicate, which is where I sourced from.